Add color picker for syntax classification

ryzngard opened the pull request:
Syntax classification and the settings for it aren't really easy to find in Tools > Options > Fonts and Colors. This introduces an easier way to set colors by following the syntax tree. If a TextSpan is classified (using our public API for classification) then we use it to get the VS color settings for that classification and display to the user, allowing it to be changed. 

Most of the code changes are adding a color picker since I couldn't find one in VS or WPF readily. Heavily motivated from https://github.com/dsafa/wpf-color-picker 

Fonts and colors settings ported with help from @JoeRobich 

# Blue Theme Example

![classification_themed_blue_mode](https://user-images.githubusercontent.com/475144/92529141-da8e1600-f1de-11ea-960e-aabb7acb937c.gif)

# Dark Theme Example

![classification_themed_dark_mode](https://user-images.githubusercontent.com/475144/92529117-d3ff9e80-f1de-11ea-9e48-a9e08a21b345.gif)

JoeRobich commented on src/VisualStudio.Roslyn.SDK/SyntaxVisualizer/Roslyn.SyntaxVisualizer.Control/FontsAndColorsHelper.cs:
```diff
@@ -0,0 +1,125 @@
+
+                // Since the color type isn't default then it has been customized, we will
+                // perform an additional check for RGB colors to see if the customized color
+                // matches the color scheme color.
```
This comment can be removed.

JoeRobich commented on src/VisualStudio.Roslyn.SDK/SyntaxVisualizer/Roslyn.SyntaxVisualizer.Control/FontsAndColorsHelper.cs:
```diff
@@ -0,0 +1,125 @@
+                return;
+            }
+
+            // Open Text Editor category for readonly access.
```
This comment needs updating.

JoeRobich reviewed (approved):
lgtm. just some stale comments

sharwell commented on src/VisualStudio.Roslyn.SDK/SyntaxVisualizer/Roslyn.SyntaxVisualizer.Control/Roslyn.SyntaxVisualizer.Control.csproj:
```diff
@@ -15,6 +15,7 @@
     <PackageReference Include="Microsoft.CodeAnalysis.Common" Version="$(MicrosoftCodeAnalysisCommonVersionVS2019)" />
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="$(MicrosoftCodeAnalysisCSharpVersionVS2019)" />
     <PackageReference Include="Microsoft.CodeAnalysis.VisualBasic" Version="$(MicrosoftCodeAnalysisVisualBasicVersionVS2019)" />
+    <PackageReference Include="Microsoft.CodeAnalysis.Workspaces.Common" Version="3.3.1" />
```
```suggestion
    <PackageReference Include="Microsoft.CodeAnalysis.Workspaces.Common" Version="$(MicrosoftCodeAnalysisWorkspacesCommonVersionVS2019)" />
```

sharwell reviewed (changes requested):
Marking Request Changes so we have an opportunity to verify two things:

1. Does the new dialog need to itself be dark-themed when the dark theme is enabled?
2. What is the accessibility state and story?

ryzngard commented:
@sharwell  good call on themeing. I updated the UI to be themed and updated the gifs in the summary. Accessibility I'm following up on with the accessibility champ

ryzngard commented:
Merging. @sharwell  this has undergone initial accessibility testing and UX review. There may be additions to the UI in a further review, but this has passed.  Will do a manual test pass with team after checkin per Abhitej's request
